### DISCUSSION TITLE
Regarding dark theme

### DISCUSSION THREAD

**Discussion by jessej-samuel** [daniel-jebarson/web-chat-app#3] [Q&A] [answered]
Hi @daniel-jebarson ! Right now it's only possible to set the theme after login. Can you make it global? Like even without login?

**Comment by daniel-jebarson** [comment]
> Hi @daniel-jebarson ! Right now it's only possible to set the theme after login. Can you make it global? Like even without login?
Yes it's a nice idea. I will do it don't worry (:

**Comment by daniel-jebarson** [comment, marked as answer]
It has been fixed on this commit https://github.com/daniel-jebarson/web-chat-app/commit/e894dc1b03e7fe3becfe573c0fd228c472d53021.
Please check it out (:

**Reply by jessej-samuel** [reply]
noice
